requirements.txt

```diff
@@ -1,3 +1,4 @@
 requests==2.24.0
 click==7.1.2
 GitPython==3.1.3
+mdutils==1.2.2
```

Comment:
In my opinion, it isn't necessary to use some util to work with `md` files if you are just going to write a readme. I suggest having a template file, let's say `basic.md` with the contents as follows:
```md
# Overview 
  
Welcome to {name}, a {type} project made using [lazystarter](https://github.com/Zenahr/lazystarter), the static site generator for people who can't be bothered.

# Dependencies
{generate_dependencies} 
```
and parse this file and replace the placeholders within the `{}` with the appropriate things.